src/SmartTransactionsController.test.ts

```diff
@@ -349,9 +350,51 @@ describe('SmartTransactionsController', () => {
           },
         },
       });
+      // update to check the persist value
+      expect(controller.metadata.smartTransactionsState.persist).toBe(true);
     });
   });
 
+  it('persists smart transactions state across browser sessions', async () => {
```

Comment:
This test is unnecessary because it doesn’t actually verify persists; instead, it focuses on whether the initial state is correctly saved in the Controller State.